src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -155,13 +155,20 @@ exports[`ProfileCallTreeView with JS Allocations matches the snapshot for JS all
       />
       <span
         class="treeViewHeaderColumn treeViewFixedColumn total"
-        title="The \\"total size\\" includes a summary of all of the bytes allocated or deallocated while this function was observed to be on the stack. This includes both the bytes where the function was actually running, and the bytes of the callers from this function."
+        title="The \\"total size\\" includes a summary of all of the bytes allocated or
+deallocated while this function was observed to be on the stack. This
+includes both the bytes where the function was actually running, and the
+bytes of the callers from this function."
```

Comment:
So I'm not happy with these snapshot changes, but it looks like ftl file is always keeping the new lines for multiple strings (at least I couldn't find a way to make them one line)